background/storage.js

```diff
@@ -238,6 +262,34 @@ function saveStyle(style) {
   }
   let existed;
   let codeIsUpdated;
+
+  if (style.usercss) {
+    return Promise.resolve().then(() => {
```

Comment:
1. all code inside uses promises so the initial Promise.resolve() is redundant and rather misleading (it kinda implies some path may be synchronous).
2. I don't like when the rarely used stuff stays in the main function flow. It's better to extract the entire contents of `if` into a function and put it before `function decide()`